src/Data/Arrays.php

```diff
@@ -0,0 +1,439 @@
+<?php
+
+/**
+ * This file is part of the Zest Framework.
+ *
+ * @author   Muhammad Umer Farooq <lablnet01@gmail.com>
+ * @author-profile https://www.facebook.com/Muhammadumerfarooq01/
+ *
+ * For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ *
+ * @since 3.0.0
+ *
+ * @license MIT
+ */
+
+namespace Zest\Data;
+
+use Zest\Contracts\Data\Arrays as ArraysContract;
+
+class Arrays implements ArraysContract
+{
+    /**
+     * Determine is given value is really, array?.
+     *
+     * @param mixed $value Value to be checked.
+     *
+     * @since 3.0.0
+     *
+     * @return bool
+     */
+    public static function isReallyArray($value)
+    {
+        return is_array($value) && !empty($value);
```

Comment:
Please consider the following code snippet:

```php
return is_array($value) && count($value) !== 0;
```

This will be more clear than original.

The real array is to check whether the `$value` is array type and its length is not be zero.